app/src/TicketEdit.js

```diff
@@ -61,21 +62,29 @@ class TicketEdit extends Component {
   async handleSubmit(event) {
     event.preventDefault();
     const { ticket } = this.state;
+    const {id } = this.state.id;
```

Comment:
This can be combined with the line above into this: 

`const { ticket, id } = this.state;`